IVAXOR.PatreonNET/Services/API/PatreonAPIv1.cs

```diff
@@ -16,18 +16,14 @@ public class PatreonAPIv1 : IPatreonAPIv1
     protected string Url { get; } = "https://patreon.com/api";
     protected HttpClient HttpClient { get; }
     protected IPatreonTokenManager TokenManager { get; }
-    protected JsonSerializerOptions JsonSerializerOptions { get; }
+    protected JsonSerializerOptions JsonSerializerOptions { get; } = PatreonJsonConstants.DefaultJsonSerializerOptions;
```

Comment:
Is this meant to be mutable?

If so, you use the appropriate constructor to copy it, rather than letting users modify your default settings through a derived class.

If not, I think it should be removed entirely.

Reply:
Yes, it's meant to be mutable. There are 2 constructors with and without `JsonSerializerOptions`.
By default, it will be set to `PatreonJsonConstants.DefaultJsonSerializerOptions` and utilize strict policy for JSON parsing and newly added metadata-based mode source-generation.
It can be overridden via second constructor if needed. For example, strict JSON parsing policy can be disabled in case of breaking V2 API changes.

Reply:
I mean that should not be using a shared instance here because the options class is mutable.

Reply:
You are right. Despite being `internal` it's still mutable. Now `PatreonJsonConstants.DefaultJsonSerializerOptions` will not be shared